### PR TITLE
Predefined images

### LINKED ISSUE #14: Have an icon or picture for each entry
In order to quickly select the right entry or simply view the associated OTP in a faster way it would be nice to have a user selectable little icon/picture for each line.

### PR DESCRIPTION
**Pull request by RichyHBM**
Fixes #14 includes images mentioned in #73

### CODE REVIEW

**Comment by flocke** [comment]
Thank you, looks good on first glance.
I'm gonna check it out in detail later today.

**Comment by flocke** [comment]
I'm not done with reviewing all your commits, but something I noticed is that the drawable folder gets very confusing now.
Maybe we could prefix all thumbnails with `thumb_` instead of `ic_` to make it a little more organized.

**Comment by flocke** [comment]
2nd thing I noticed while testing: if you search the thumbnail selection and click on the filtered results it does not apply the thumbnail. It still shows the letter image afterwards. In the un-filtered selection everything works fine.

**Comment by RichyHBM** [comment]
Hmm, that's odd, I'll look into that :S

**Comment by flocke** [comment]
Other than those two points everything looks fine and is working nicely on my phone.

File: app/src/main/java/org/shadowice/flocke/andotp/Utilities/DatabaseHelper.java [hunk 1]
```diff
@@ -89,14 +89,15 @@ public static String entriesToString(ArrayList<Entry> entries) {
         return json.toString();
     }
 
-    public static ArrayList<Entry> stringToEntries(String data) {
+    public static ArrayList<Entry> stringToEntries(Context context, String data) {
```
Comment (flocke) [inline app/src/main/java/org/shadowice/flocke/andotp/Utilities/DatabaseHelper.java]: Why did you add the Context here? It is never used.

**Comment by RichyHBM** [comment]
I think that's left over from when I was storing the thumbnail in the entry class, and as such it needed the context to construct one. Will remove it once I'm home, however that may not be until tonight so fell free to remove it if you want to merge it sooner

**Comment by flocke** [comment]
Then I'm gonna fix it myself and merge this now ;)
